src/universe/index.ts

```diff
@@ -1534,7 +1534,12 @@ export class Universe extends APICarrier {
 
   public async messageTemplates (): Promise<messageTemplate.MessageTemplate[] | undefined> {
     try {
-      const res = await this.http.getClient().get(`${this.universeBase}/${messageTemplate.MessageTemplates.endpoint}`)
+      const res = await this.http.getClient().get(`${this.universeBase}/${messageTemplate.MessageTemplates.endpoint}`, {
+        // Fetch only none-FlowBuilder templates (That are of kind "AutomationEngine")
+        params: {
+          kind: ['Generic', 'ProductGeneral', 'ProductDetail']
+        }
+      })
```

Comment:
Generally LGTM but: 

1. I'd probably offer some options for the SDK user (overwrite params etc)
2. If you want, you can also refactor this to our newer list fetching implementation (which would allow for paginating from the UI)

Reply:
The UI doesn't allow pagination and it would severely increase the scope to have pagination designed and implemented on the front-end aswell. So I think this can wait.

I considered having the user pass params, but my brief discussion with the flow builder team indicated that there's no probable use case where these AutomationEngine templates are utilized by anything other than the engine, so I thought of keeping it simple and minimally intrusive.